src/Sinch/Conversation/Apps/Credentials/BasicAuthCredentials.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Text;
+
+namespace Sinch.Conversation.Apps.Credentials
+{   /// <summary>
+    ///     It consists of a username and a password.
+    /// </summary>
+    public sealed class BasicAuthCredential
+    {
+        /// <summary>
+        ///     Basic auth password.
+        /// </summary>
+#if NET7_0_OR_GREATER
+        public required string Password { get; set; }
+#else
+        public string Password { get; set; }
+#endif
+        
+
+        /// <summary>
+        ///     Basic auth username.
+        /// </summary>
+#if NET7_0_OR_GREATER
+        public required string Username { get; set; }
+#else
+        public string Username { get; set; }
+#endif
+        
+
+        /// <summary>
+        ///     Returns the string presentation of the object
+        /// </summary>
+        /// <returns>String presentation of the object</returns>
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
+            sb.Append("class BasicAuthCredential {\n");
+            sb.Append("  Password: ").Append(Password).Append("\n");
```

Comment:
Should we offer this critical information ?
This mean we could found clear password in any layer using a print/tostring onto such instance (log files, ...)

Reply:
Good point. Will be removed.